lib/resources/r4/care_plan.yaml

```diff
@@ -0,0 +1,14 @@
+---
+name: CarePlan
+field_name_base_url: http://hl7.org/fhir/R4/careplan-definitions.html#CarePlan
+fields:
+- name: category
+  required: 'No'
+  type: List of CodeableConcept
+  binding:
+    description: >
+      Identifies what "kind" of plan this is to support differentiation between multiple co-existing plans; e.g. "Home health", "psychiatric", "asthma", "disease management", etc.
```

Comment:
Can add this This is also used to return a CareTeam (CarePlan with participants). "assess-plan"